src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -1447,6 +1490,8 @@ def ramp_fit_overall(
     log.debug('The execution time in seconds: %f', tstop - tstart)
 
     # Compute the 2D variances due to Poisson and read noise
+    sum_in_var_p2 = s_inv_var_p3.sum(axis=0)
+    wh_zero = np.where(sum_in_var_p2 == 0.)
```

Comment:
I don't see that wh_zero is used.

Reply:
These two lines were debugging lines of code that I was using for analysis.  I will remove them.